easybuild/easyconfigs/t/TTK/TTK-0.9.8.9-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'TTK'
+version = '0.9.8.9'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://topology-tool-kit.github.io"
+description = "The Topology ToolKit (TTK) is an open-source library and software collection for topological data analysis in scientific visualization."
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/topology-tool-kit/%(namelower)s/archive']
+sources = ["%(version)s.tar.gz"]
+checksums = ['d0cd86bbea9e761b0a9acb3ce871e5faaec17ee86c608fe6f472aa21399c31f5']
+
+dependencies = [
+    ('Boost', '1.66.0'),
+    ('Python', '3.6.4'),
+    ('VTK', '8.2.0', versionsuffix),
```

Comment:
You can make any modifications in your local installation and add an additional version of VTK to `2018a`. However, we restrict dependencies in the central repository of easyconfigs to one per toolchain to keep a manageable maintenance cost. In the case of `foss/2018a` the version of VTK available is `8.1.0`. Therefore, this easyconfig of TTK should use that dependency to be accepted.

Reply:
This breaks python bindings of TTK. In that, the bindings are successfully generated only with VTK >= 8.2.

Reply:
In such a case, if you want to contribute this easyconfig for `TTK`, please consider moving it to either `foss/2019a`, `foss/2019b` or `foss/2020a`. All those toolchains provide VTK v8.2.

Reply:
@lexming, I've now removed the offending EB files.